src/Service/OrbitService.cpp

```diff
@@ -150,6 +150,12 @@ int OrbitService::Run(std::atomic<bool>* exit_requested) {
   }
   grpc_server->AddCaptureStartStopListener(producer_side_server.get());
 
+  // The client is looking for the "READY" keyword to learn whether the service finish its start up
+  // and is ready to accept a connection. Check out the ServiceDeployManager on how the detection
+  // works.
+  std::puts("\n--- READY to accept connections! ---\n");
```

Comment:
Honestly if the client expects "READY" I think that this should just be "READY", it's a bit confusing otherwise.

Reply:
My intention was to add a log message that doesn't mess with our usual stderr log output for users that start OrbitService on the command line. Printing exactly the string that is detected on the client side would mean we don't add a line break. And that definitely messes up the log output. But I'm not sure how important that is.

Another option: I could add a command line flag and only output the `READY` keyword when the command line flag is present.

What would be your preferred way?

Reply:
I was missing that this gets printed on the command line. If it's just about the line break, another option would be to only print "\nREADY\n". But there can be value in having the verbose string that you have now.

> I could add a command line flag and only output the READY keyword when the command line flag is present.

Naah, I don't think that's necessary.

Reply:
Okay, I think I'm fine going with `\nREADY\n`. Thank you!

Reply:
Changed!